ts/utils/util.py

```diff
@@ -38,6 +53,26 @@ def list_classes_from_module(module, parent_class=None):
     return classes
 
 
+def load_compiler_config(config_file_path):
+    """
+    Load a compiler {compiler_name -> compiler }
+    Can be extended to also support kwargs for ONNX and TensorRT
+    """
+    if not os.path.isfile(config_file_path):
+        logger.warning(f"{config_file_path} is missing. PT 2.0 will not be used")
```

Comment:
Should we use info here as users who do not provide the file might be confused by the warning?